pkg/workflow/controller/handlers/workflowtrigger/cron_job.go

```diff
@@ -91,17 +92,9 @@ func (t *CronTrigger) getKeyFromTrigger() string {
 
 // Run triggers the workflows.
 func (t *CronTrigger) Run() {
-	if t.WorkflowRun.Labels == nil {
-		t.WorkflowRun.Labels = make(map[string]string)
-	}
-	t.WorkflowRun.Labels[common.WorkflowNameLabelName] = t.WorkflowRun.Spec.WorkflowRef.Name
-	if t.WorkflowRun.Annotations == nil {
-		t.WorkflowRun.Annotations = make(map[string]string)
-	}
-	t.WorkflowRun.Annotations[s_common.AnnotationTrigger] = s_common.CronTimerTrigger
-
 	for {
 		t.WorkflowRun.Name = fmt.Sprintf("%s-%s", t.WorkflowTriggerName, rand.String(5))
+		t.WorkflowRun.Annotations[s_common.AnnotationAlias] = t.WorkflowRun.Name
```

Comment:
Is the annotations here certain not `nil`?

Reply:
yeah, it is ensured here https://github.com/caicloud/cyclone/pull/871/files#diff-b5d86b4920ce755c97d6cf00d2daaefeR132